packages/next/build/index.ts

```diff
@@ -550,7 +550,9 @@ export default async function build(
       const pageKeys = {
         pages: Object.keys(mappedPages),
         app: mappedAppPages
-          ? Object.keys(mappedAppPages).map((key) => normalizeAppPath(key))
+          ? Object.keys(mappedAppPages).map(
+              (key) => normalizeAppPath(key) || '/'
```

Comment:
This should probably be explicitly checking for `''` instead of using `||`

Reply:
We do have a couple of other places doing the same thing, it should probably be built into `normalizeAppPath`.